content/components/overview/PendingReviews.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { Col, List, Panel, Row, Tag } from "rsuite";
+
+import { RecipeV3 } from "devtools/types/recipes";
+import { chunkBy } from "devtools/utils/helpers";
+
+export const PendingReviews: React.FC<any> = (props: {
+  data: Array<RecipeV3>;
+}) => {
+  const { data } = props;
+  const pendingReviews = data.filter((recipe) => {
+    const {
+      latest_revision: { approval_request },
+    } = recipe;
+    return approval_request && approval_request.approved === null;
```

Comment:
We can use fancy things:

```suggestion
    return recipe.latest_revision.approval_request?.approved === null;
```

Reply:
oooh fancy